internal/app/go-filecoin/internal/submodule/syncer_submodule.go

```diff
@@ -57,16 +56,15 @@ type nodeSyncDispatcher interface {
 }
 
 type nodeChainSelector interface {
-	NewWeight(context.Context, block.TipSet, cid.Cid) (uint64, error)
 	Weight(context.Context, block.TipSet, cid.Cid) (uint64, error)
 	IsHeavier(ctx context.Context, a, b block.TipSet, aStateID, bStateID cid.Cid) (bool, error)
 }
 
 // NewSyncerSubmodule creates a new chain submodule.
-func NewSyncerSubmodule(ctx context.Context, config syncerConfig, repo chainRepo, blockstore *BlockstoreSubmodule, network *NetworkSubmodule, discovery *DiscoverySubmodule, chn *ChainSubmodule, pvt *version.ProtocolVersionTable) (SyncerSubmodule, error) {
+func NewSyncerSubmodule(ctx context.Context, config syncerConfig, repo chainRepo, blockstore *BlockstoreSubmodule, network *NetworkSubmodule, discovery *DiscoverySubmodule, chn *ChainSubmodule) (SyncerSubmodule, error) {
 	// setup block validation
 	// TODO when #2961 is resolved do the needful here.
-	blkValid := consensus.NewDefaultBlockValidator(config.BlockTime(), config.Clock(), pvt)
+	blkValid := consensus.NewDefaultBlockValidator(config.BlockTime(), config.Clock())
```

Comment:
I probably would have left the pvt in to aid future upgrades. Removing it also make sense, though, so this is fine.